.azure-pipelines/steps/set-success-for-stages.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
```

Comment:
This file isn't used. And I assume the no-op pipeline calling steps/update-github-status.yml should work, so we don't need this file ?

Reply:
Oops, yeah, that was a first-iteration attempt 🙂 Deleted in https://github.com/DataDog/dd-trace-dotnet/pull/2334/commits/b7fb1f7974a9508383860fb612c75ab0f80d6906